Homework13/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Homework13.Data.Constants;
+using Homework13.PageObjects;
+using NUnit.Framework;
+
+namespace Homework13.Tests
+{
+    public class WebTableTests : BaseTest
+    {
+        const string firstName = "Rita";
+        const string lastName = "Bunets";
+        const string email = "email@example.com";
+        const string age = "26";
+        const string salary = "100000";
+        const string department = "QA";
+        const string entryValue = "Rita Bunets 26 email@example.com 100000 QA  ";
```

Comment:
I don't like you get expected value such way (+ space at the end of string). Anyway after fix of comment above you will fix it

Reply:
fixed